lpc17prl.lua

```diff
@@ -172,6 +174,32 @@ if opts.mode == 'read' then
 end
 
 
+if opts.mode == 'calculate' then
+  if not opts.fname then D.abort(2, "Binary file not provided") end
+
+  local f, err = io.open(opts.fname, "r+b")
```

Comment:
Usually you see such order for open-mode.

```suggestion
  local f, err = io.open(opts.fname, "rb")
```

but what you wrote seems to be good as well :)